src/styles/index.scss

```diff
@@ -1,3 +1,275 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
 }
+
+.catalog {
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+}
+
+/* #region-Header */
+$linkColor: #060b35;
+$activ: #00acdc;
+
+.nav {
+  display: flex;
+  background: #fff;
+  height: 60px;
+  justify-content: flex-end;
+  align-items: center;
+  font-size: 12px;
+  text-transform: uppercase;
+  gap: 20px;
+}
+
+.nav :last-child {
+  margin-right: 50px;
+}
```

Comment:
The selector `.nav :last-child` is using a descendant combinator, which might not work as expected if there are nested elements within `.nav`. Consider using `.nav > :last-child` if you intend to target the last direct child of `.nav`.